pkg/controllers/disruption/controller.go

```diff
@@ -242,3 +245,20 @@ func (c *Controller) logAbnormalRuns(ctx context.Context) {
 		}
 	}
 }
+
+// logInvalidBudgets will log if there are any invalid schedules detected
+func (c *Controller) logInvalidBudgets(ctx context.Context) {
+	nodePoolList, err := nodepoolutil.List(ctx, c.kubeClient)
+	if err != nil {
+		logging.FromContext(ctx).Debugf("listing nodepools, %s", err)
+	}
+	var buf bytes.Buffer
+	for _, np := range nodePoolList.Items {
+		if _, err := np.GetAllowedDisruptions(ctx, c.clock, 100); err != nil {
```

Comment:
nit: Maybe a comment here to say that you are sending through a dummy value to check the validity of the disruption; otherwise, I'm left guessing where 100 came from